avro2tf/src/main/scala/com/linkedin/avro2tf/parsers/Avro2TFJobParamsParser.scala

```diff
@@ -49,7 +49,8 @@ case class Avro2TFParams(
   partitionFieldName: String,
   termOnlyFeatureList: Boolean,
   discardUnknownEntries: Boolean,
-  enableFilterZero: Boolean
+  enableFilterZero: Boolean,
+  passThrough:Boolean
```

Comment:
Do you consider to have another job to do pass through, for example, Avro2TFDataWriter instead of having passThrough option

Reply:
Yes, A separate job is fine. However, to easily utilize the new job, we also need to make changes in our gradle plugin, which i think is not necessary for this simple feature support.

Reply:
changing template is fine. My concern is if passThrough is true, you don't need any of other config params at all (I think that's why you make json input from require to optional). But on the other hand, it looses check. For example, if passthrough is false, you do need to provide json config. Let's not create a monolithic job with multiple branches and only use config to control it. What do you think?

Reply:
Many parameters are shared, i don't adding such a simple logic makes the job complicated. I do add check if passthrough mode is not enabled. Also changing the template is not that easy. It would take a least two full days to add a new template. For such simple feature support, adding a new template is really not necessary.

Reply:
I see. In that sense (it shares a lot of other params), I'm fine putting them in a job.